pyproject.toml

```
@@ -0,0 +1,18 @@
+[build-system]
+requires = [
+  # Essentials
+  "setuptools>=40.6.0",
+  "wheel",
+
+  # Plugins
+  "setuptools_scm[toml] >= 3.5",
```

Let's document the motivation here:
```suggestion
  "setuptools_scm[toml] >= 3.5",  # version is required for "no-local-version" scheme + toml extra is needed for supporting config in this file
```